src/common/translation/german-translation-set.ts

```diff
@@ -345,4 +345,8 @@ export const germanTranslationSet: TranslationSet = {
 
     controlPanel: "Systemsteuerung",
     controlPanelSettingsDescription: "Dieses Plugin erlaubt es dir, Elemente der Systemsteuerung schnell zu finden.",
+
+    reminder: "Erinnerung",
+    reminderPrefix: "Präfix",
+    reminderSettingsDescription: "Mit diesem Plugin können Sie Erinnerungen festlegen\n",
```

Comment:
Why is `\n` neede here?

Reply:
Good catch!
It seems that when i copied and pasted from Google Translate it brought it with it.
I will fix it